config/plugins/interactive_environments/jupyter/config/jupyter.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE interactive_environment SYSTEM "../../interactive_environments.dtd">
+<interactive_environment name="Jupyter (Programming Environment)">
```

Comment:
I want to improve usability a little bit and Jupyter does not say anything to the user what he can except. Even worse R devs don't know about Jupyter and would not try it maybe. So I would like to indicate what it actually does. I like @jmchilton idea "Jupyter (Scripting Environment)" but there is also an IGo kernel so we can add it in the future ... this would not be `scripting` anymore, right?